docs/paper/dev/getting-started/project-setup/mcdev-project-setup.md

```diff
@@ -0,0 +1,72 @@
+---
+slug: /dev/getting-started/project-setup/mcdev-project-setup
+description: Step-by-step instructions on how to setup a plugin with the MinecraftDev plugin.
```

Comment:
I think clarifying the two “plugin”s is needed

Reply:
Something like this?
```suggestion
description: Step-by-step instructions on how to setup a plugin with the MinecraftDev IntelliJ plugin.
```

Reply:
maybe change the first plugin to project and keep that Intelljj

Reply:
Ok I changed it. Will push when I have more feedback.